server/app/views/NorthStarBaseView.java

```diff
@@ -75,6 +75,7 @@ protected ThymeleafModule.PlayThymeleafContext createThymeleafContext(
         "civicEntityFullName", settingsManifest.getWhitelabelCivicEntityFullName(request).get());
     context.setVariable("adminLoginUrl", routes.LoginController.adminLogin().url());
     context.setVariable("closeIcon", Icons.CLOSE);
+    context.setVariable("checkCircleIcon", assetsFinder.path("Images/uswds/icon-check-circle.svg"));
```

Comment:
(blocking) This is going to become difficult to manage. [See my comments](https://github.com/civiform/civiform/issues/8349#issuecomment-2323041177) over at the related issue.